Created logical turtle post with embedded code

EAkita opened the pull request:
First version. I'll make some changes to it later today

EAkita committed:
Created logical turtle post with embedded code 

First version. I'll make some changes to it later today

jasfromnz commented on _posts/eakita/2020-08-26-logical_turtle-eakita.md:
```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+author: eakita
```
In the examples author comes first, above layout- not sure if this will make a difference.

jasfromnz reviewed (commented):
Looks good so far!

EAkita committed:
Added reflection. Updated embedded link. 

Moved author on top of layout to reflect review changes. Edited embedded link to reflect updated code with variables, conditionals, user input and operators, Added reflection also. I believe it doesn't matter much though.

jasfromnz reviewed (approved):
Final version looks good!
